content/en/docs/tasks/administer-cluster/migrating-from-dockershim/topics-on-dockershim-and-cri-compatible-runtimes.md

```diff
@@ -0,0 +1,27 @@
+---
+reviewers:
+- tbd
+title: Topics on dockershim and CRI-compatible runtimes
+content_type: concept
+weight: 20
+---
+<!-- overview -->
+The following topics cover the dockershim deprecation and CRI-compatible runtimes."
+<!-- body -->
+
+## Primary sources
+
+[https://github.com/kubernetes/enhancements/issues/2221](Removing dockershim from kubelet #2221)
+
+[https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2221-remove-dockershim/README.md](KEP-2221: Removing dockershim from kubelet)
+
+[Kubernetes Blog: Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+
+[Kubernetes Documentation: Migrating from dockershim](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/)
+
+[Kubernetes Documentation: Container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/)
+
+
+## Secondary sources
+
+* [Docker blog: What developers need to know about Docker, Docker Engine, and Kubernetes v1.20](https://www.docker.com/blog/what-developers-need-to-know-about-docker-docker-engine-and-kubernetes-v1-20/)
```

Comment:
For consistency:
- Remove the bullet here, or
- Add bullets for the `Primary sources` entries